### PR TITLE
Config files to deploy to OpenShift

### PR DESCRIPTION
**Pull request by johnbradley**
Adds files to deploy this shiny app on an OpenShift cluster consisting of
- Files describing OpenShift compatible R Docker images 
- OpenShift templates to build and deploy the docker images

## Docker files
Files in the `base-shiny` directory are configuration to create a base Docker image containing all the requirements for running this application. This uses R version 4.0.2 from the [rocker/shiny-verse:4.0.2](https://github.com/rocker-org/rocker-versioned2) repo.
A top level `Dockerfile` adds the current R code to the `base-shiny` image to create our application image.

## OpenShift files
Files in the `openshift` directory create the following items
- 5 GB storage volume to hold user data
- Build to create an image named `base-shiny` to install requirements including [Shiny_install.R](https://github.com/Duke-GCB/shiny_PD/blob/master/Shiny_install.R)
- Build to create an image named `spd-shiny-app` to run the application based on `base-shiny`
- Deployment that will run pods to run the application using the `spd-shiny-app` image

## Remaining Issues
- I used the name `spd` just based on the initials of this project. If there is a better name for it I would like to update the config files to more accurately represent this project.
- The `Shiny_Startup_v1.R` files references `Shiny_Tables_v1.R` which does not currently exist in this repo.
https://github.com/gmw12/shiny_PD/blob/5c974bc537229992436c8a0caeea45eb2eb10aff/Shiny_Startup_v1.R#L76

I created a test deployment in our OpenShift cluster after removing the line which sources `Shiny_Tables_v1.R`.
The app runs and displays options to select data but the app may break due to not sourcing Shiny_Tables_v1.R.
Another issue is the app also only lets you select from data stored within the pods instead of uploading.
If you would like I can email you the URL to the test deployment.

Any feedback is welcomed.

Thanks.

### CODE REVIEW

**Comment by johnbradley** [comment]
Screenshot of the main screen running in OpenShift:

Screen Shot 2020-07-17 at 12 12 23 PM

Example file browser that is looking at the files within the pod(think VM):
Screen Shot 2020-07-17 at 12 12 34 PM

**Comment by gmw12** [comment]
Shiny_Tables_v1 added to the repo

**Comment by gmw12** [comment]
Please email the URL - thanks!

File: openshift/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+ddh-app
```
Comment (hlapp) [inline openshift/README.md]: Probably not _ddh-app_?

**Pull request merged**